test/mpfr.jl

```diff
@@ -921,3 +921,5 @@ end
         @test to_string(big"-1.0") == "-1.0"
     end
 end
+
+@test isapprox(beta(big(1.0),big(1.2)), beta(1.0,1.2), rtol=eps())
```

Comment:
```jl
@test beta(big(1.0),big(1.2)) ≈ beta(1.0,1.2) rtol=eps()
```

Reply:
`rtol=eps()` seems rather stringent, because it assumes that our `beta(::Float64, ::Float64)` function is exactly rounded.  I'm not sure we want to require that.  Maybe `eps()*4`?

Reply:
I had no idea that was a thing!